knowledge_runtime/knowledge_runtime/core/config.py

```
@@ -0,0 +1,24 @@
+
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+
+class Settings(BaseSettings):
+    PROJECT_NAME: str = "Knowledge Runtime"
+    VERSION: str = "1.0.0"
+    ENABLE_API_DOCS: bool = True
+    ENVIRONMENT: str = "development"
+    HOST: str = "0.0.0.0"
+    PORT: int = 8200
+    INTERNAL_SERVICE_TOKEN: str = "knowledge-runtime-token"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "INTERNAL_SERVICE_TOKEN"

(S105)